data/pigui/libs/forwarded.lua

```diff
@@ -22,6 +22,11 @@ ui.getWindowContentSize = pigui.GetWindowContentSize ---@type fun(name: string):
 ui.setNextWindowPos = pigui.SetNextWindowPos ---@type fun(pos: Vector2, cond: string, pivot: Vector2?)
 ui.setNextWindowSize = pigui.SetNextWindowSize ---@type fun(size: Vector2, cond: string)
 ui.setNextWindowSizeConstraints = pigui.SetNextWindowSizeConstraints ---@type fun(min: Vector2, max: Vector2)
+--- Collapse or expand the next window
+---@param collapse boolean|nil If not set or true, collapse the next window, else expand it 
+function ui.setNextWindowCollapsed( collapse )
+    pigui.SetNextWindowCollapsed( collapse )
+end
```

Comment:
The addition of an extra function call here is unnecessary at best. I strongly recommend defining this as:
```suggestion
--- Collapse or expand the next window
ui.setNextWindowCollapsed = pigui.SetNextWindowCollapsed ---@type fun(collapse: boolean?)
```

We prefer to define all "wrapping functions" which add extra logic in addition to the exported functions in `data/pigui/libs/wrappers.lua` or a topic-specific file.